docs/DeveloperGuide.md

```diff
@@ -348,6 +348,30 @@ Throughout the onboarding guide, Overlays and Highlights are used to direct the
 ##### The `Overlay` Class
 The Overlay class is implemented using 2 translucent panes binded to the top and bottom of the OnboardingWindow. This makes it possible to create an desired area of focus by leaving only an area uncovered.
 
+### Markdown-like Text Processing
+The `TextStyleHelper` class contains a text processor that processes common text modifications into UI elements. This is mainly used in the `ResultDisplay` UI component, where it shows text output each time a command is executed. In particular, it is used in the error messages, in order to present error messages better,
+
+Example of styled text:
+![StyledTextExample](images/StyledTextExample.png)
+
+It can be easily used in other components of ContaX if the need arises.
+
+It currently supports:
+- Italics
+  - Enclosing text with `*`
+- Bold
+  - Enclosing text with `**`
+- Bold and Italics
+  - Enclosing text with `***`
+- Monospaced
+  - Enclosing text with `` ` ``
+
+This was done by changing the `ResultDisplay` to use a `TextFlow` object, and dynamically generate a List of styled `Text` objects to add to the `ResultDisplay`.
```

Comment:
```suggestion
This was done by changing the `ResultDisplay` to use a `TextFlow` object, and dynamically generating a List of styled `Text` objects to add to the `ResultDisplay`.
```
Small nitpick on grammar

Reply:
thanks for the catch, fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/251/commits/d89a7bbac3bd9d5bc369d78a3e759252f6e68df0